src/libraries/System.Runtime/tests/System/Type/TypeTests.cs

```diff
@@ -527,6 +527,7 @@ public void GetTypeByName_Invalid(string typeName, Type expectedException, bool
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtimelab/issues/155" /* NativeAot */)]
         public void GetTypeByName_InvokeViaReflection_Success()
```

Comment:
What is the problem with this one?

Reply:
I didn't debug it, but the type is not assembly qualified so we're supposed to look in the assembly that is calling the API. I assume CoreCLR knows the calling assembly even when reflection is involved. I figure this was needed for CAS, but for native AOT I assume we can put this in the "never going to happen" bucket.